app/i18n/i18n.js

```diff
@@ -5,10 +5,12 @@ import dateLocaleFR from 'date-fns/locale/fr'
 import datelocaleEN from 'date-fns/locale/en'
 import datelocaleAR from 'date-fns/locale/ar'
 import datelocaleES from 'date-fns/locale/es'
+import datelocalePTBR from 'date-fns/locale/pt-BR'
```

Comment:
It seems the `date-fns` version we're using doesn't support this, but latest definitely does: https://github.com/date-fns/date-fns/tree/master/src/locale.

We can either:
- Update `date-fns` to the latest version (if you have the time to do it)
- Or comment this line & merge with a `//TODO`, we'll be able to add it after the update

Reply:
I've answered this comment outside of the thread, my bad!